examples/using-router/pages/error.js

```diff
@@ -1,16 +1,19 @@
-import React from 'react'
+import {Component} from 'react'
 import Header from '../components/Header'
 import Router from 'next/router'
 
-const ErrorPage = ({ aa }) => (
-  <div>
-    <Header />
-    <p>This should not be rendered via SSR</p>
-  </div>
-)
+export default class extends Component {
+  static getInitialProps () {
+    console.log(Router.pathname)
```

Comment:
I believe it is intentionally broken to show what will happen when you use Router the wrong way.